### PR TITLE
Release/ publish package

### PR DESCRIPTION
**Pull request by rrahir**
## Description:

description of this task, what is implemented and why it is implemented that way.

Odoo task ID : [TASK_ID](https://www.odoo.com/web#id=TASK_ID&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_lt("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/components/spreadsheet.ts [hunk 1]
```diff
@@ -18,10 +18,9 @@ import { LinkEditor } from "./link/link_editor";
 import { SidePanel } from "./side_panel/side_panel";
 import { TopBar } from "./top_bar";
 
-const { Component, useState } = owl;
-const { useRef, useExternalListener } = owl.hooks;
-const { xml, css } = owl.tags;
-const { useSubEnv } = owl.hooks;
+const { useExternalListener, useSubEnv } = hooks;
+const useRef = hooks.useRef;
```
Comment (rrahir) [inline src/components/spreadsheet.ts:22]: to remove

File: src/components/composer/composer.ts [hunk 2]
```diff
@@ -17,9 +18,9 @@ import { ContentEditableHelper } from "./content_editable_helper";
 import { FunctionDescriptionProvider } from "./formula_assistant";
 import { Dimension } from "./grid_composer";
 
-const { Component } = owl;
-const { useRef, useState } = owl.hooks;
-const { xml, css } = owl.tags;
+const { useState } = hooks;
+const { useRef } = owl.hooks;
```
Comment (rrahir) [inline src/components/composer/composer.ts:22]: to remove

**Comment by rrahir** [comment]
@fw-bot ignore

**Comment by rrahir** [comment]
@robodoo rebase-ff

**Review by LucasLefevre — COMMENTED** [review]
robodoo r+

**Comment by LucasLefevre** [comment]
robodoo rebase-ff

**Pull request closed without merging**